jina/types/arrays/document.py

```diff
@@ -3,6 +3,7 @@
 from abc import abstractmethod
 from collections.abc import MutableSequence, Iterable as Itr
 from contextlib import nullcontext
+import numpy as np
```

Comment:
`numpy` import should come after the system imports

Reply:
done